.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,39 +1,34 @@
 ---
 name: Bug report
-about: Create an issue to report a bug for the SAM Translator
-title: 'Bug: TITLE'
+about: Report a bug concerning the AWS SAM transform
+title: ''
 labels: stage/needs-triage, type/bug
 assignees: ''
 
 ---
 
-<!-- Make sure we don't have an existing Issue that reports the bug you are seeing (both open and closed). 
-If you do find an existing Issue, re-open or add a comment to that Issue instead of creating a new one. -->
+<!-- Make sure we don't have an existing issue that reports the bug you are encountering (both open and closed).
+If you do find an existing issue, re-open or add a comment to that issue instead of creating a new one.
+Add the --debug flag to any SAM CLI commands you are running. -->
 
-### Description:
+### Description
 <!-- Briefly describe the bug you are facing.-->
 
 
+### Steps to reproduce
+<!-- Provide detailed steps to reproduce the bug. -->
 
-### Steps to reproduce:
-<!-- Provide detailed steps to replicate the bug, including steps from third party tools (CDK, etc.) -->
 
+### Observed result
+<!-- Describe what you observed. -->
 
 
-### Observed result:
-<!-- Please provide command output with `--debug` flag set.-->
+### Expected result
+<!-- Describe what you expected. -->
 
 
-
-### Expected result:
-<!-- Describe what you expected.-->
-
-
-
-### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)
+### Additional environment details
```

Comment:
adding something like "it will be helpful if you can provide a snippet of your template?" Many cases we ask for template to further investigate.

Reply:
Added to "Steps to reproduce"